config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
   devise_for :users
+  resources :foods
```

Comment:
- I feel specifying the actions for the foods resources would be nice as the requirements don't allow for `update` action. What do you think? 😁